Update stepfunctions troubleshooting doc (#19340)
DataDog/documentation: content/en/serverless/step_functions/troubleshooting.md

- In your AWS console, open your Step Function and ensure that your state machine has `"Payload.$": "States.JsonMerge($$, $, false)"` on the Lambda steps.
- Execute your Step Function once and verify that the `TaskScheduled` event log of the Lambda step has the payload containing data from the [Step Function context object][3].

> ## I can only see `aws.stepfunctions` root span but cannot see any step spans

Reviewer: ```suggestion
## I can see the `aws.stepfunctions` root span but I cannot see any step spans
```

> Please enable `Include execution data` option on the state machine's logging. By enabling this option, log execution input, data passed between states, and execution output will be logged in the logs which are then used by Datadog backend to construct these step spans for you.

Reviewer: ```suggestion
Please enable the `Include execution data` option on the state machine's logging. After enabling this option, log execution input, data passed between states, and execution output is logged. The Datadog backend uses the logs to construct these step spans for you.
```

## Some step spans are missing in the traces

> - For actions, we support basic actions of Lambda and DynamoDB, e.g. Lambda Invoke, DynamoDB GetItem, DynamoDB PutItem, DynamoDB UpdateItem... etc.

Reviewer: ```suggestion
- For actions, we support basic actions of Lambda and DynamoDB. For example, Lambda Invoke, DynamoDB GetItem, DynamoDB PutItem, DynamoDB UpdateItem and more.
```

> - For different flows, we do not support `Wait`, `Choice`, `Map`, `Success`, `Fail`, and `Pass`. For `Parrallel` flow, you would be able to see parallel executing spans stacking on top of each other, but there will be no `Parrallel` spans showing on the flamegraph.

Reviewer 2: ```suggestion
- For different flows, we do not support `Wait`, `Choice`, `Map`, `Success`, `Fail`, and `Pass`. For `Parallel` flow, you would be able to see parallel executing spans stacking on top of each other, but there will be no `Parallel` spans showing on the flame graph.
```
